src/debug-adapter/debugEventManager.ts

```diff
@@ -98,7 +102,7 @@ export class DebugEventManager {
     panelViewInput: PanelViewInput
   ): Promise<void> {
     let isNewAndObject = false;
-    const id = variable.value;
+    const id = variable.value === 'null' ? variable.value : `${DebugEventManager.objectPrefix}${variable.value}`;
```

Comment:
Is the prefix (none or `DebugEventManager.objectPrefix`) always consistent with `isNewAndObject` (1/2)?

Reply:
No, It can be an existing object